gulpfile.js

```diff
@@ -111,29 +111,53 @@ gulp.task('render-datasource-templates', function() {
         if (filename.match(/\.ejs$/)) {
             var templateFilename = path.join(sourceDir, filename);
             var template = fs.readFileSync(templateFilename,'utf8');
-            var result = ejs.render(template, null, {filename: templateFilename}), result_big='';
+            var result = ejs.render(template, null, {filename: templateFilename});
 
             // Remove all new lines. This means you can add newlines to help keep source files manageable, without breaking your JSON.
             // If you want actual new lines displayed somewhere, you should probably use <br/> if it's HTML, or \n\n if it's Markdown.
             result = result.replace(/(?:\r\n|\r|\n)/g, '');
 
             var outFilename = filename.replace('.ejs', '.json');
-
+            var resultJson = '', resultJson_big = '';
             try {
-                result = JSON.stringify(JSON5.parse(result), null, 0);
-                result_big = JSON.stringify(JSON5.parse(result), null, 2);
+                resultJson = JSON.stringify(JSON5.parse(result), null, 0);
+                resultJson_big = JSON.stringify(JSON5.parse(result), null, 2);
                 console.log('Rendered template ' + outFilename);
             } catch (e) {
-                console.warn('Warning: Rendered template ' + outFilename + ' is not valid JSON');
+                if (e.name === 'SyntaxError') {
+                    var context = 20;
+                    console.error('Syntax error while processing templates: ' + e.message);
+                    console.error(result.substring(e.at - context, e.at + context));
+                    console.error(new Array(context + 1).join('-').substring(0, Math.min(e.at, context)) + '^');
+                }
+                console.warn('Warning: Rendered template ' + outFilename + ' is not valid JSON.');
             }
-            fs.writeFileSync(path.join(targetDir, outFilename), new Buffer(result));
+            fs.writeFileSync(path.join(targetDir, outFilename), new Buffer(resultJson));
             // write a non-minified version too.
-            fs.writeFileSync(path.join(targetDir, filename.replace('.ejs', '_big.json')), new Buffer(result_big));
+            fs.writeFileSync(path.join(targetDir, filename.replace('.ejs', '_big.json')), new Buffer(resultJson_big));
         }
     });
 
 });
 
 gulp.task('watch-datasource-templates', ['render-datasource-templates'], function() {
     return gulp.watch(['datasources/**/*.ejs','datasources/*.json'], watchOptions, [ 'render-datasource-templates' ]);
+});
+
+// Regenerate the anti-LGA filter in datasources/includes/lga_filter.ejs
+// Needs to be run manually every now and then.
+gulp.task('update-lga-filter', function() {
```

Comment:
Release meaning the `release` task?  It's a bit odd that that task is the same as `build` except it _doesn't_ include `list-ga-services`.  Maybe we should just get rid of it entirely?

I think it's worth doing it automatically, otherwise we're pretty likely to forget to run it.

Getting tasks to run in order is a huge PITA in gulp, but I'll work on that.